### PR TITLE
Add caution for httpProxy

### PR DESCRIPTION
**Pull request by futuretea**
Signed-off-by: futuretea <Hang.Yu@suse.com>

**Related issue**
https://github.com/harvester/harvester/issues/3172

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/advanced/settings.md [hunk 1]
```diff
@@ -135,6 +135,12 @@ https://172.16.0.1/v3/import/w6tp7dgwjj549l88pr7xmxb4x6m54v5kcplvhbp9vv2wzqrrjhr
 - HTTPS 请求的代理 URL：`"httpsProxy": "https://<username>:<pswd>@<ip>:<port>"`
 - 主机名和/或 CIDR 的逗号分隔列表：`"noProxy": "<hostname | CIDR>"`
 
+:::caution
+
+如果你配置了 `httpProxy` 和 `httpsProxy`, 那么你必须同时将Harvester节点的CIDR填入 `noProxy`, 否则Harvester集群将会崩溃。
```
Comment (vickyhella) [inline i18n/zh/docusaurus-plugin-content-docs/current/advanced/settings.md:140]:
```suggestion
如果你配置了 `httpProxy` 和 `httpsProxy`, 那么你必须同时将 Harvester 节点的 CIDR 填入 `noProxy`, 否则 Harvester 集群将会崩溃。
```

**Review by vickyhella — APPROVED** [review]
Just a small suggestion.

File: i18n/zh/docusaurus-plugin-content-docs/version-v1.0/settings/settings.md [hunk 2]
```diff
@@ -77,6 +77,12 @@ https://172.16.0.1/v3/import/w6tp7dgwjj549l88pr7xmxb4x6m54v5kcplvhbp9vv2wzqrrjhr
 - HTTPS 请求的代理 URL：`"httpsProxy": "https://<username>:<pswd>@<ip>:<port>"`
 - 主机名和/或 CIDR 的逗号分隔列表：`"noProxy": "<hostname | CIDR>"`
 
+:::caution
+
+如果你配置了 `httpProxy` 和 `httpsProxy`, 那么你必须同时将Harvester节点的CIDR填入 `noProxy`, 否则Harvester集群将会崩溃。
```
Comment (vickyhella) [inline i18n/zh/docusaurus-plugin-content-docs/version-v1.0/settings/settings.md:82]:
```suggestion
如果你配置了 `httpProxy` 和 `httpsProxy`, 那么你必须同时将 Harvester 节点的 CIDR 填入 `noProxy`, 否则 Harvester 集群将会崩溃。
```

**Review by guangbochen — APPROVED** [review]
LGTM, and let's squash the commit, thanks.

**Comment by futuretea** [comment]
squashed

**Pull request merged**